test/integration/connection-monitoring-and-pooling/connection.test.js

```diff
@@ -162,24 +162,8 @@ describe('Connection', function () {
     let testClient;
 
     afterEach(async () => {
-      let savedError;
-      if (client) {
-        try {
-          await client.close();
-        } catch (err) {
-          savedError = err;
-        }
-      }
-      if (testClient) {
-        try {
-          await testClient.close();
-        } catch (err) {
-          savedError = err;
-        }
-      }
-      if (savedError) {
-        throw savedError;
-      }
+      if (client) await client.close();
```

Comment:
I think it's something we can be okay with, close generally doesn't error but if it does, while inconvenient as it might hang the test run at the end I don't want to change this to swallow the error entirely. And then capturing the error from each and raising if one exists seems overly complex. Also if close does error the run may hang anyway so we're in a pretty bad spot if we get there.